doc/mosfire_howto.rst

```diff
@@ -0,0 +1,548 @@
+    m121128_0107.fits |              lampoffflats |         7.8 |        45.0 |       unknown | K-spectroscopy | ic348_TK_M03A |     1,1 | 56259.14262084 | 1.41291034 |  14.5479 |       K |        off |    none |    none |     0.0 |     107 |     0 |      -1 |     -1
+    m121128_0112.fits | pixelflat,illumflat,trace |         7.8 |        45.0 |       unknown | K-spectroscopy | ic348_TK_M03A |     1,1 | 56259.14425684 | 1.41291034 |  14.5479 |       K |         on |    none |    none |     0.0 |     112 |     0 |      -1 |     -1
+    m121128_0113.fits | pixelflat,illumflat,trace |         7.8 |        45.0 |       unknown | K-spectroscopy | ic348_TK_M03A |     1,1 | 56259.14450569 | 1.41291034 |  14.5479 |       K |         on |    none |    none |     0.0 |     113 |     0 |      -1 |     -1
+    m121128_0114.fits | pixelflat,illumflat,trace |         7.8 |        45.0 |       unknown | K-spectroscopy | ic348_TK_M03A |     1,1 | 56259.14479678 | 1.41291034 |  14.5479 |       K |         on |    none |    none |     0.0 |     114 |     0 |      -1 |     -1
+    data end
+
+Inspecting this file, we want to make sure that all the frame types were accurately assigned in the
+:ref:`pypeit_file:Data Block`. If not, we can make edits using the instructions in
+:ref:`pypeit_file:Edits to the Data Block`. We can also remove any bad (or undesired) calibration
+or science frames from the list, by either deleting them altogether or commenting out with a ``#``.
+
+In this example, all the frametypes were accurately assigned. However, as mentioned earlier, we use the
+OH lines in science frames for the wavelength calibration, therefore we do not want to keep the arc frames
+(*m121128_0119.fits*, *m121128_0120.fits*) in the Data Block list, and we comment them out.
+
+.. tip::
+    If the user wants to use the arc frames instead, they can keep the 2 arc frames in the list, but need
+    to edit the ``frametype`` for the science frames (*m121128_0214.fits - m121128_0217.fits*), i.e., remove
+    the ``arc`` and ``tilt`` frame type. In addition, the changes explained in :ref:`mosfire_wavecalib` will have to
+    be added to the :ref:`pypeit_file:Parameter Block`.
+
+Other possible edits to the :ref:`pypeit_file:Data Block` are related to the ``comb_id`` and ``bkg_id``
+columns, which instruct PypeIt on the desired frame combination and background subtraction.
+For Keck/MOSFIRE data, PypeIt tries to automatically set the ``comb_id`` and ``bkg_id`` using the dither
+information (reported here in the ``dithpat``, ``dithpos``, and ``dithoff`` columns) recorded in the header
+of the science frames (see :ref:`mosfire_combid_bkgid`); however,
+the user can edit these columns according to the preferred reduction (see :ref:`a-b_differencing` and
+:ref:`2d_combine` for more info).
+
+Finally, in this example, we also edit the :ref:`pypeit_file:Parameter Block` adding the following lines::
```

Comment:
I recommend using
```
... the following lines:

.. code-block:: ini

...
```
to get the syntax highlighting.

Reply:
Done. Thank you! 
I have a question though. I got the colors a bit off when I compile it on my machine (one line is green and one red, see below). Do you get the same thing? Am I doing something wrong?

<img width="375" alt="Screen Shot 2022-10-19 at 2 46 28 PM" src="https://user-images.githubusercontent.com/16786072/196830786-61c92ebd-726e-474b-b61f-3b5370e052ba.png">

Reply:
Interesting.  Maybe try upgrading sphinx?  It might also be the indentation?  Sphinx can sometimes be super picky that you're, e.g., using 4 spaces for each tab.  If you display `_build/html/pypeit_par.html#instrument-specific-default-configuration`, does that have all the variable names as green and the values as red?

Reply:
I think you are right. It's a rendering problem in my machine. I get the same problem with colors in other docs that I did not edit and that look OK in the PypeIt webpage. I'll try re-installing Sphinx. Thanks!